db/schema.rb

```diff
@@ -465,6 +465,12 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "picture"
+    t.boolean  "payed",                default: false
+    t.boolean  "swags",                default: false
```

Comment:
If we have `has_banner`, then it makes sense to name this `has_swag`, for naming uniformity purposes.